app/src/main/java/com/sopt/now/MainActivity.kt

```diff
@@ -1,15 +1,26 @@
 package com.sopt.now
 
+import android.content.Intent
 import android.os.Bundle
 import androidx.activity.enableEdgeToEdge
 import androidx.appcompat.app.AppCompatActivity
 import androidx.core.view.ViewCompat
 import androidx.core.view.WindowInsetsCompat
+import com.sopt.now.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
+    lateinit var binding : ActivityMainBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
 
+        val id = intent.getStringExtra("id")
+        val nickname = intent.getStringExtra("nickname")
+        val mbti = intent.getStringExtra("mbti")
+
+        binding.tvMainNickname.setText(nickname)
+        binding.tvMainIdcontent.setText(id)
+        binding.tvMainMbticontent.setText(mbti)
```

Comment:
```suggestion
        with(binding){
        tvMainNickname.setText(nickname)
        tvMainIdcontent.setText(id)
        tvMainMbticontent.setText(mbti)
        }
```
scope함수라는 것을 사용하게 되면 binidng을 위에서 한번만 써도 되요! scope함수에 대해 찾아보는것도 좋아용

Reply:
scope로 묶는거 저는 좋아해요 ㅎㅎ 한번에 휘잡을 수 있다니,, 매력적이지 않나효..? 이 부분도 함수로 빼주시면 좋을 것 같습니다! 아래 링크를 참고하면 좋을 것 같네요

https://medium.com/@fatihcoskun/kotlin-scoping-functions-apply-vs-with-let-also-run-816e4efb75f5

Reply:
scope 함수 너무 좋아보이는데요?!?!??!?!?? 당장 사용하겠습니다 알려주셔서 감사합니다❤️